salt/metalk8s/container-engine/containerd/installed.sls

```diff
@@ -36,6 +37,8 @@ Install containerd:
       {%- if grains['os_family'].lower() == 'redhat' %}
       - metalk8s_package_manager: Install container-selinux
       {%- endif %}
+    - watch_in:
+      - service: Ensure containerd running
```

Comment:
Similar as in other places, will the service be *re*started when the package is upgraded?

Reply:
Yes